test/libraries/Pool.t.sol

```diff
@@ -76,14 +81,17 @@ contract PoolTest is Test {
         state.modifyLiquidity(params);
     }
 
-    function testSwap(uint160 sqrtPriceX96, uint24 swapFee, Pool.SwapParams memory params) public {
+    function testSwap(uint160 sqrtPriceX96, uint24 swapFee, uint24 protocolFee, Pool.SwapParams memory params) public {
         // Assumptions tested in PoolManager.t.sol
         params.tickSpacing = int24(bound(params.tickSpacing, TickMath.MIN_TICK_SPACING, TickMath.MAX_TICK_SPACING));
-        swapFee = uint24(bound(swapFee, 0, 999999));
+        swapFee = uint24(bound(swapFee, 0, 1000000));
+        protocolFee = uint24(bound(protocolFee, 0, 0x3E83E8));
```

Comment:
i see the magic hex is `(1000 << 12) | 1000`, but we should use the MAX_FEE_BOTH_TOKENS constant here for readability

Reply:
👍